_source/_posts/2020-06-22-deploy-dotnet-container-aws-fargate.md

```diff
@@ -3,112 +3,99 @@ disqus_thread_id: 8089923701
 ---
 
-In the last article I wrote, I showed you [how to host a serverless .NET application using AWS Lambda](/blog/2020/06/08/serverless-lambda-functions-csharp). In the article, we talked about the history of serverless and how companies are using these types of technology to simplify the process of delivering APIs and functionality faster than traditional methods. There are some problems that will start to arise in this type of application when you need more capability than your standard HTTP protocols GET, POST, PUT, DELETE, etc. A great example of this is Web Sockets.
+In the last article I wrote, we learned [how to host a serverless .NET application using AWS Lambda](/blog/2020/06/08/serverless-lambda-functions-csharp). In the article, we talked about the history of serverless and how companies are using these types of technology to simplify delivering APIs and functionality faster than traditional methods. Some problems will arise in this type of application when you need more capability than your standard HTTP requests like GET, POST, PUT, DELETE, etc. A great example of this is Web Sockets.
 
-## Understanding Containers vs Virtual Machines
+{% include toc.md %}
 
-Around 2013, a product was released called [Docker](https://www.docker.com/), and with it began a shift of how we think about hosting applications and managing infrastructure. Infrastructure teams and operations have been leveraging Virtual Machines with great success for decades, so you may be tempted to think "isn't this just a virtual machine?" In spirit, yes, in application no.
+## Understanding Containers vs. Virtual Machines
 
-To talk about containers we have to talk about Virtual Machines. A virtual machine is a virtual representation of a computer all the way from its boot process to loading a full operating system. This provides a lot of flexibility because you get to have fine-grained control over an entire virtual computer, but it has all the pitfalls of a full computer as well. The Operating System has to be maintained, patched, and updated; disks have to be managed, and all manner of operational overhead goes into their care and feeding. Notably, they are also very large because of the fact that an entire OS has to go on them. So when you start a Virtual Machine you have to go through the entire boot sequence of a traditional computer.
+Around 2013, [Docker](https://www.docker.com/) was released, and with it began a shift in how we think about hosting applications and managing infrastructure. Infrastructure teams and operations have been leveraging Virtual Machines with great success for decades, so you may be tempted to think, "isn't this just a virtual machine?" In spirit, yes, in application no.
```

Comment:
```suggestion
Around 2013, [Docker](https://www.docker.com/) was released, beginning a shift in how we think about hosting applications and managing infrastructure. Infrastructure teams and operations have been leveraging Virtual Machines with great success for decades, so you may be tempted to think, "isn't this just a virtual machine?" In spirit, yes; in application, no.
```
simplifying a strong opening sentence. adding semi-colon to separate the point/counterpoint